apps/renterd/components/Config/Recommendations.tsx

```diff
@@ -177,7 +177,9 @@ export function Recommendations() {
             {usableHostsCurrent}/{hostTarget50}
           </Text>
           <Text size="16" weight="medium">
-            {recommendations.length === 1
+            {recommendations.length === 0
+              ? 'No recommendations'
+              : recommendations.length === 1
               ? '1 recommendation'
               : `${recommendations.length} recommendations`}{' '}
```

Comment:
Here's another option that avoids the double ternary, in case you hadn't considered it:

```tsx
  {recommendations.length === 0 ? 'No recommendations' : `${recommendations.length} recommendation${recommendations.length > 1 ? 's' : ''}`}
```

You could also pull that example out into a `const` for readability, but what's above works well too! Approving regardless--just thought I'd share a different approach.

Reply:
@telestrial agreed that this could be cleaned up, I think the full solution is to add a really slick pluralize helper to use in all cases like this, follow up PR: https://github.com/SiaFoundation/web/pull/706